var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
This change probably isn't necessary.

Reply:
Okay I make sure to remove unnecessary chances and sure hope the decision of using the "g" prefix gets approved, I will make sure to commit the unnecessary changes removed.